esx_service/cli/vmdkops_admin.py

```diff
@@ -955,9 +958,13 @@ def tenant_access_set(args):
     if args.volume_totalsize:
         volume_totalsize_in_MB = convert.convert_to_MB(args.volume_totalsize)
 
+    if not validate_create_value(str(args.allow_create)):
```

Comment:
Ok. And this explicit value from VMODL API can in wrong/invalid not a boolean value always. So we need a validation in tenant_access_add as well. Am I correct?